setup/setup-ci/security-scanner/tests/amlsecscan.json

```diff
@@ -0,0 +1,3 @@
+{
+    "logAnalyticsResourceId": "/subscriptions/23782349-c970-4549-acbb-7ed9d00ebb32/resourceGroups/DefaultResourceGroup-WUS2/providers/Microsoft.OperationalInsights/workspaces/AmlDev"
```

Comment:
Are you good with exposing these resource URIs?

Reply:
Updated with generic placeholders